src/frontend/Sider/Behandling/RevurderFra/TilsynBarn/VedtakshistorikkTilsynBarnTabellVisning.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { Table } from '@navikt/ds-react';
+
+import { erVedtakInnvilgelse, VedtakBarnetilsyn } from '../../../../typer/vedtak/vedtakTilsynBarn';
+import { formaterNullableIsoDato } from '../../../../utils/dato';
+
+type Props = {
+    vedtakBarnetilsyn: VedtakBarnetilsyn;
+};
+
+const VedtakshistorikkTilsynBarnTabellVisning: React.FC<Props> = ({ vedtakBarnetilsyn }) => {
+    if (!erVedtakInnvilgelse(vedtakBarnetilsyn)) return 'Ingen vedtakshistorikk';
+    if (vedtakBarnetilsyn?.beregningsresultat?.vedtaksperioder === undefined)
+        return 'Ingen vedtakshistorikk';
```

Comment:
For litt bedre lesbarhet så kan det være fint med curly brackets her? 

```suggestion
    if (vedtakBarnetilsyn?.beregningsresultat?.vedtaksperioder === undefined) {
        return 'Ingen vedtakshistorikk';
     }   
       
```

Reply:
Fixed: be38e86c41a5c71da358703fc159e541411bbbc5